domain/src/test/java/org/oppia/android/domain/translation/TranslationControllerTest.kt

```diff
@@ -1306,6 +1313,7 @@ class TranslationControllerTest {
   private fun createAppLanguageSelection(language: OppiaLanguage): AppLanguageSelection {
     return AppLanguageSelection.newBuilder().apply {
       selectedLanguage = language
+      selectedLanguageValue = language.number
```

Comment:
For this & the change below, why are these needed?

Reply:
Reverted this change.